c++/src/kj/string.h

```diff
@@ -76,6 +79,8 @@ class StringPtr {
   inline StringPtr(const char* begin KJ_LIFETIMEBOUND, const char* end KJ_LIFETIMEBOUND): StringPtr(begin, end - begin) {}
   inline StringPtr(String&& value KJ_LIFETIMEBOUND) : StringPtr(value) {}
   inline StringPtr(const String& value KJ_LIFETIMEBOUND);
+  inline StringPtr(const ConstString& value KJ_LIFETIMEBOUND);
+  constexpr StringPtr(const LiteralStringPtr& ptr);
```

Comment:
Should this be defined as an operator on `LiteralStringPtr`? I don't know what difference exists between `operator T()` vs `T(const U&)`

Reply:
You don't actually need either one because LiteralStringPtr is a subclass of StringPtr so the existing copy constructor for StringPtr will work fine.